.travis.yml

```diff
@@ -88,7 +88,7 @@ script:
     - EB_BOOTSTRAP_VERSION=$(grep '^EB_BOOTSTRAP_VERSION' $TRAVIS_BUILD_DIR/easybuild/scripts/bootstrap_eb.py | sed 's/[^0-9.]//g')
     - EB_BOOTSTRAP_MD5SUM=$(md5sum $TRAVIS_BUILD_DIR/easybuild/scripts/bootstrap_eb.py | cut -f1 -d' ')
     - EB_BOOTSTRAP_FOUND="$EB_BOOTSTRAP_VERSION $EB_BOOTSTRAP_MD5SUM"
-    - EB_BOOTSTRAP_EXPECTED="20170503.01 fdf8ca02b6600c4ce0b6a02b88f1f40a"
+    - EB_BOOTSTRAP_EXPECTED="20170705.01 90d2dd008be027c35c1d194cbf5b1d44"
```

Comment:
Sure. I don't think it matters much, this is just a mechanism just help with not forgetting to bump the version of the bootstrap script (as opposed to actually verifying a checksum), but it doesn't hurt either. Done in d28755d7.